src/modules/appbuilder.js

```diff
@@ -0,0 +1,55 @@
+import Template from './obarr.js';
+import localGet from './localstorage';
+
+const todoListContainer = document.querySelector('.listItem');
+
+const displayList = () => {
+  todoListContainer.innerHTML = '';
+  localGet.forEach((item, id) => {
+    todoListContainer.innerHTML
+    += `
+      <div class='toDoItem'>
+        <input class='item' id='check-${id}',
+         completed)' type='checkbox' value=${item.completed}>
+        <input type='text' class='findInput' id='input-${id}' value=${item.description} />
+        <i onclick='updateList(${id})'  id='options-${id}'></i>
+        <i onclick='removeList(${id})' class='fa-solid fa-trash' id='delete-${id}'></i>
+      </div>
+    `;
+  });
+};
+
+const addList = (description, completed, index) => {
+  const listAdded = new Template(description, completed, index);
+  localGet.push(listAdded);
+  localStorage.setItem('listStorage', JSON.stringify(localGet));
+  displayList();
+};
```

Comment:
- [ ] OPTIONAL You can add a condition for not adding the empty title tasks - you will avoid empty entrees by users 👍 
![image](https://user-images.githubusercontent.com/49064106/194301208-5fd624b4-bd31-4813-a771-85cf4cf7c98c.png)